content/v2.0/monitor-alert/templates/AWS.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Monitor Amazon Web Services (AWS)
+seotitle: Monitor Amazon Web Services (AWS)
+description: >
+  Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
+menu:
+  v2_0:
+    parent: Monitor with templates
+weight: 101
+---
+
+Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
```

Comment:
- Link "AWS CloudWatch Monitoring template" to the template on GitHub.
- Link all the AWS products to their AWS landing pages:
    - https://aws.amazon.com/
    - https://aws.amazon.com/ec2/
    - https://aws.amazon.com/elasticloadbalancing/
    - https://aws.amazon.com/cloudwatch/